### PR TITLE
Header and AlbumList

### PR DESCRIPTION
**Pull request by benkutdev**

### CODE REVIEW

File: components/AlbumList/Albumlist.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    { id: 1, src: curtisAlbum },
+  ];
+
+  return (
+    <AlbumListContainer>
+      {albumCovers.map((album) => (
+        <Image
+          alt={album.id}
+          src={album.src}
+          key={album.id}
```
Comment (Roland-Hufnagel) [inline components/AlbumList/Albumlist.js:57]: the key has to be unique.

File: components/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import Image from "next/image";
+import styled from "styled-components";
+import recordLogo from "../../public/images/greenvinyl.png";
+
+const Header = styled.header`
+  position: relative;
```
Comment (Roland-Hufnagel) [inline components/Header/Header.js:6]: You did not match the acceptance-criteria: 'The header is fixed'

File: components/Header/Header.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      <Image
+        src={recordLogo}
+        alt="vinyl-round-logo"
+        title="vinyl-record"
+        height={63}
+        width={63}
+        style={{
+          position: "absolute",
+          top: 4,
+          left: 4,
```
Comment (Roland-Hufnagel) [inline components/Header/Header.js:26]:
```suggestion
          top: "4px",
          left: "4px",
```

File: pages/_app.js [hunk 4]
```diff
@@ -1,8 +1,16 @@
 import GlobalStyle from "../styles";
+import { createGlobalStyle } from "styled-components";
+
+const TheLook = createGlobalStyle`
+  body {
+    background-color: #145A32;
+  }
+`;
```
Comment (Roland-Hufnagel) [inline pages/_app.js:8]: You should better define your global style (body, background-color) in the GlobalStyle-component.

File: pages/index.js [hunk 5]
```diff
@@ -1,7 +1,13 @@
-export default function HomePage() {
+import AppHeader from "../components/Header/Header.js";
+import AlbumList from "../components/AlbumList/Albumlist.js";
```
Comment (Roland-Hufnagel) [inline pages/index.js:2]:
```suggestion
import Albumlist from "../components/AlbumList/Albumlist.js";
```

**Review by Roland-Hufnagel — CHANGES_REQUESTED** [review]
This looks already fine. We left some comments. Please have a look at it.

File: styles.js [hunk 6]
```diff
@@ -10,5 +10,7 @@ export default createGlobalStyle`
   body {
     margin: 0;
     font-family: system-ui;
+    background-color: #145A32;
+
```
Comment (ThomasKirst) [inline styles.js:14]: Empty line can be removed, check your prettier settings (format on save).

**Review by ThomasKirst — APPROVED** [review]
Just found an empty blank line which you can remove.
Keep up the good work 👍

**Pull request merged**